parcels/kernels/TEOSseawaterdensity.py

```diff
@@ -0,0 +1,90 @@
+"""Collection of pre-built sea water density kernels"""
```

Comment:
You've copied this `polyTEOS10_bsq` Kernel into this new file, but haven't removed the old `seawaterdensity.pu` file?